user_docs/en/userGuide.t2t

```diff
@@ -2785,10 +2785,19 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 
 ++ Log Viewer ++[LogViewer]
 The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.
-Using NVDA+F1 will open the log viewer and display developer information about the current navigator object.
 
 Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it shows the most recent output since the Log viewer was opened.

```

Comment:
```suggestion
Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it loads new log output generated after the Log viewer was opened.

```

Reply:
@XLTechie, I guess your point was that that the log viewer does not show only the newly logged items when pressing F5, but the whole log including newly logged items, right?

An alternative phrasing could be:

```suggestion
Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer to append to its content the new log output generated after the Log viewer was opened.
```

Reply:
@CyrilleB79 Yes. Here's another potential way of saying it, which preserves brevity by about four words. :)

```suggestion
Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it appends log entries generated after you opened the viewer.
```